lib/daru/dataframe.rb

```diff
@@ -2229,7 +2229,11 @@ def transpose
     end
 
     # Pretty print in a nice table format for the command line (irb/pry/iruby)
-    def inspect spacing=10, threshold=15
+    #
+    # @option [Boolean] all: when true, will remove thresholding (and will override any value set for threshold)
+    def inspect spacing=10, threshold=15, all: false
```

Comment:
I think we should have a method that can set the threshold for Daru::DataFrame and Vector to display maximum number of rows and columns. 

So I believe we should have  a method like 

```
Daru::DataFrame.inspect_max_rows = nil # to load all the rows

```
Default value will be set to 15.

What do you think ? @paisible-wanderer

Reply:
In that case, you can mention the number of rows in the threshold option, itself.